### DISCUSSION TITLE
Possible rebrand of WingetUI.

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#1561] [General]
Dear users and contributors,

WingetUI has evolved into supporting different package managers, not only Winget, leading to the name **WingetUI** confusing new users, and not representing the main point of this application. Every day I am observing more often users getting confused by WingetUI and winget, which is an expectable situation taking into account the little difference between the two names.

Thhe new WingetUI name will be **UnigetUI** (proposed by @Spacellary)

### Please see [this post](https://github.com/marticliment/WingetUI/discussions/1561#discussioncomment-8695791) for updated information

~~This is why I have been thinking seriously about a rebrand, involving a new name and icon that do represent more accurately what WingetUI wants to be: **An interface to manage your packages**. The rebrand would involve an integral redesign of the iconography (shortcuts, executables and system tray icons) and banners (website, social and installer banners), and changing the old logo for a new logo that represents an actual package, and with a more fluent design (following W11 design guidelines).~~

~~I am aware that this change would cause even more confusion on top of the one already caused by the current name, but I think it must be done, to prevent further issues and misunderstandings from happening in the future.~~

The rebrand would be applied gradually, warning users before the roll out of the new name and icon. For a period of time, the app would be named `NewAppName (formerly WingetUI)` to allow users to adapt to the new name scheme.

What do you think? Do you have suggestions?

**Comment by ppvnf** [comment]
I like the idea of Packages. I am not sure if the name being a common word would make it harder for people to discover the project, for example if you type WingetUI on Google you can find the project, however typing Packages wouldn't (because Google doesn't know you are looking for an app). Some suggestions: MultiPackages, OmniPackages, PackagesUI, myPackages, winPackages
About the logo, I think it looks nice, but a little bit similar to MS Powerpoint logo IMO: <img width="16" alt="pwppoint" src="https://github.com/marticliment/WingetUI/assets/73800734/582b2633-aa85-4536-8e02-4a7f6a90b998">

**Reply by mapi68** [reply]
I totally agree with @ppvnf.
For image, I like something like this with name of system packages out of the box:

<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/HLk813Wn/Package-01-1024x725.jpg' border='0' alt='Package-01-1024x725'/></a>

**Reply by dcog989** [reply]
That's most likely not true - e.g. type in just 'files' and Google / Bing / Duck assumes you are looking for the app of that name.

**Comment by itayasaf1990xd** [comment]
The new icon is really good!!! but the name should be PackagesUI and not just Packages.

I also think that you should add more fluent design elements and maybe some tooltips for icons or add text next to the icon when there is available space.

**Comment by Shak1990** [comment]
PackagesUI sounds like a better name that truly represents the app.

About the new Sketch of a possible new icon, I do not think it looks or feels Fluent enough it feels 3D.

**Comment by Spacellary** [comment]
I also like PackagesUI - "The all-in-one UI for your favorite package managers."

**Comment by SynapseScribe** [comment]
i agree the suggested logo looks like MS powerpoint or publisher

wingetUI grew so much as I consider it more than either an updater/installer for apps or libraries/scripts/modules repository - it is a complete package manager. I wouldn't call it other than PackageManagerUI

**Comment by dcog989** [comment]
Agree that the project should be rebranded. 'Packages' is meh / OK. 'PackagesUI' - imagine Steve saying "We're calling it the AdvancedSmartphone phone".

I'd choose:

**Bundle** - _a collection of things wrapped together_.

No other product, software or otherwise, using that name.

**Comment by ak1dd** [comment]
Branding should also take into consideration brand visibility and discoverability.  A web search - or even a github search for "packages" will yield tons of results and likely bury WingetUI.  Conversations about the utility also get confusing.   I understand your dilemna but I would have never found or used WingetUI if I wasn't specifically searching for what it does for the most used windows package manager.

Might I suggest "Everything-Everywhere-Up-to-date"!  :)  EEUTD.

**Reply by Spacellary** [reply]
I was gonna say something like "Universal Package Manager UI - UPMUI" but it's not really **really** universal yet so maybe some people don't like that. Honestly I'm way too used to WingetUI, it's hard to come up with something else as catchy... Maybe GPT 4.5 can help.

**Reply by itayasaf1990xd** [reply]
I agree this is why packagesUI could be the best name because UI from the previous name stays and just the starter word is changing to a more generic and better-describing word of the software.

**Comment by Biyakuga** [comment]
I think that the name PackagesUI would be a better fit for the app as it aligns with its intended purpose. Additionally, I suggest that you modify the new icon as it doesn't look or feel Fluent enough.
You could consider looking for a designer on Reddit or other forums who would be willing to assist you for free.

P.S. Thank you for your hard work!

**Comment by Stephan-P** [comment]
About translation:
- Do you want the app name to be translated?
- If not, then we would need a clear distinction in Tolgee between Packages (the app name) and Packages as in the plural of a package

**Reply by marticliment** [reply]
The app name IMO should be kept in english. I will figure this out. I will figure out how can we handle this

**Comment by marticliment** [comment]
Dear contributors, 

Firrst of all, thank you for your suggestions. They have guided me in the path of finding a new appropiate name for WingetUI, that better represents its capabilities. After reading all of your ideas, and asking ChatGPT for some more suggestions, I have three final candidates, and I would like to hear your opinions.
 - **PackagesUI**: <i>Like Packages, but without the potential issue of the Google penalty for using an already existing word</i>
 - **PackgetUI**: <i>Maintains the visual structure of WingetUI, as well as the latest getUI piece.</i>
 - **PKGS**: <i>Breaks with the old name, gives it a more modern-sounding name.</i>

Furthermore, the rebranding won't be implemented in WingetUI 3.0. Rather, a banner will be shown to the user, explaining the transition to the new name. WingetUI will then be effectively rebranded on the next (3.1?) release. 
To not further confuse the users, the icon will remain the same (at least for now).

Thank you for your help and support!

**Reply by dcog989** [reply]
Meh. 

I wonder if Google would have succeeded if Larry and Sergei had called it 'AdvancedInternetSearchEngine.com'?

'Bundle' was the best suggestion. ;)

**Reply by SynapseScribe** [reply]
it feels more natural for the next major version to have the name changed (v3.0). i'd rather show the banner in the last v2 minor version

the name should be unique. there are already projects with these names:
packagesUI - https://packagecontrol.io/packages/PackagesUI
PKGS - https://pkgs.org/

PackgetUI is available, but the app shouldn't relate just to winget, since it manages scoop, chocolatey, npm etc as well.
I previously suggested PackageManagerUI, however unity also has something similar https://docs.unity3d.com/Packages/com.unity.package-manager-ui@2.0/manual/index.html

What about inserting Win in the name, since it only applies for windows OS ?

**Reply by itayking1990** [reply]
PackagesUI sounds the best and is the least confusing choice!
Users already know WingetUI but because you handle multiple packages and create UI for them therefore Winget gets replaced with packages.
Maybe you can say only PackageUI without the "s" for plural if the name has already taken

**Reply by Spacellary** [reply]
> PackagesUI

Is the more clear sounding one but I think it suffers from already being used too much as mentioned.

If we're going for the _getUI_, maybe three can me more options like *AllgetUI*, *AnygetUI*... Or adding the word Manager and Packages, such as *AllgetManager*, *AllgetPackages*, *AnygetManager*, *AnygetPackages*, *AnygetSoftware*...

---

There's also **"UnigetUI"** which does deliver the message and is closer to the original naming and sound.

"UnigetUI - Your *near* universal package manager consolidation UI *for Windows*."

*Who knows if it becomes truly Universal and cross platform one day ;)*

*Realistically all names will have slightly overlap with existing projects but I think the functionality of your project with the UI first approach is a distinctive feature.*

**Reply by Spacellary** [reply]
> Furthermore, the rebranding won't be implemented in WingetUI 3.0. 

I think a banner _now_ and rebranding already in 3.0 would have the biggest impact and have bigger perceived value and impact of the new branding too.

**Comment by Lowfeye** [comment]
Like Uj947nXmRqV2nRaWshKtHzTvckUUpD said, since it only works with Windows OS, how about **WinPackUI**

**Comment by marticliment** [comment]
Dear contributors, thank you for your replies. After hearing your suggestions, and after finding already existing projects with some of the already proposed suggestions, the new name will be the one proposed by @Spacellary:

# **UnigetUI**

This is the banner that will be shown to the users. It will be shipped with WingetUI 3.0, expected to be released next week.

![image](https://github.com/marticliment/WingetUI/assets/53119851/c43b7abe-c6f6-404b-8187-4795fa148eed)


As a remainder, the rebranding won't be implemented in WingetUI 3.0. Rather, the banner will be shown to the user, explaining the transition to the new name. WingetUI will then be effectively rebranded on the next (3.1? 3.0.1?) release. 
To not further confuse the users, the icon will remain the same (at least for now).

Thank you for your help and support!
